packages/stable/src/api/userProfiles/profilesApi.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 Cognite AS
+
+import { BaseResourceAPI } from '@cognite/sdk-core';
+import {
+  UserProfileItem,
+  UserIdentifier,
+  UserProfilesSearchRequest,
+  UserProfilesListResponse,
+} from '../../types';
+
+export class ProfilesAPI extends BaseResourceAPI<UserProfileItem> {
+  /**
+   * [Get the user profile of the principal issuing the request](https://api-docs.cognite.com/20230101/tag/User-profiles/paths/~1profiles~1me/get/)
+   *
+   * ```js
+   * const response = await client.profiles.me();
+   * ```
+   */
+  public me = async (): Promise<UserProfileItem> => {
+    const path = this.url('me');
+
+    const response = await this.get<UserProfileItem>(path);
+    return this.addToMapAndReturn(response.data, response);
+  };
+
+  /**
+   * [List all user profiles](https://api-docs.cognite.com/20230101/tag/User-profiles/paths/~1profiles/get)
+   *
+   * ```js
+   * const response = await client.profiles.list({ limit: 1000, nextCursor: 'abc' });
+   * ```
+   */
+  public list = (scope?: {
+    limit?: number;
+    cursor?: string;
+  }): UserProfilesListResponse => {
+    return super.listEndpoint(this.callListEndpointWithGet, scope);
+  };
+
+  /**
+   * [Retrieve one or more user profiles by ID](https://api-docs.cognite.com/20230101/tag/User-profiles/paths/~1profiles~1byids/post)
+   *
+   * ```js
+   * const response = await client.profiles.retrieve([{ userIdentifier: 'abcd' }]);
+   * ```
+   */
+  public retrieve = async (
+    userIdentifiers: UserIdentifier[]
+  ): Promise<UserProfileItem[]> => {
+    return super.retrieveEndpoint(userIdentifiers);
+  };
+  /**
+   * [Search user profiles](https://api-docs.cognite.com/20230101/tag/User-profiles/paths/~1profiles~1search/post)
```

Comment:
the reason I ask, is that they won't appear as code snippets in the portal otherwise, example:

<img width="1026" alt="Screenshot 2023-11-28 at 16 03 05" src="https://github.com/cognitedata/cognite-sdk-js/assets/10908415/83a287cb-fc46-4576-963f-62157f7340d2">

Reply:
Good point, I guess I should modify this file? https://github.com/cognitedata/service-contracts/blob/master/versions/v1/user_profiles.yml

The current link is being referenced by others, e.g. the Python SDK. What's the best way of communicating to everyone that the link will break? @polomani

Reply:
it might not break, as there is a chance that both will work no? All other resources use operationids anyway, it's better to be consistent.
Also it should be possible to update the link in python sdk

Reply:
Made a PR: https://github.com/cognitedata/service-contracts/pull/2388
Awaiting review from the API team

Reply:
@polomani above PR merged and urls updated 😃